resources/web/wwi/Server.js

```diff
@@ -86,19 +86,20 @@ export default class Server {
     } else if (message.indexOf('error:') === 0) {
       this.onError();
       alert('Session server ' + message);
-    } else if (message.indexOf('docker:') === 0) {
+    } else if (message.indexOf('loading:') === 0) {
+      let body = message.substring(9);
       let percent;
       if (document.getElementById('webots-progress-bar-percent'))
         percent = document.getElementById('webots-progress-bar-percent').value;
-      else if (message.startsWith('docker: Creating network'))
+      else if (body.startsWith('Creating network'))
         percent = 20;
-      else if (message.startsWith('docker: Step '))
-        percent = 20 + 65 * parseInt(message.charAt(13)) / (parseInt(message.charAt(15)) + 1);
-      else if (message.endsWith('done'))
+      else if (body.startsWith('Step '))
+        percent = 20 + 65 * parseInt(body.charAt(13)) / (parseInt(body.charAt(15)) + 1);
```

Comment:
Don't we need to update these values, e.g., 13 and 15 to something like 5 and 7 or so?